### PR TITLE
Added MQTTDisconnect packet

### PR DESCRIPTION
**Pull request by ivanv6**
Added MQTTDisconnect packet

**Checklist:**

-   [ ] If you are new to Scapy: I have checked [CONTRIBUTING.md](https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md) (esp. section submitting-pull-requests)
-   [ ] I squashed commits belonging together
-   [ ] I added unit tests or explained why they are not relevant
-   [ ] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)
-   [ ] If the PR is still not finished, please create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

fixes #xxx

### CODE REVIEW

**Comment by gpotter2** [comment]
Hi & thanks for the PR.
Could you add a unit test to https://github.com/secdev/scapy/blob/master/test/contrib/mqtt.uts
and fix the flake8 failures ? Thanks

```
scapy/contrib/mqtt.py:165:1: E305 expected 2 blank lines after class or function definition, found 1
scapy/contrib/mqtt.py:305:17: E231 missing whitespace after ','
scapy/contrib/mqtt.py:305:32: E231 missing whitespace after ','
scapy/contrib/mqtt.py:317:27: E231 missing whitespace after ','
scapy/contrib/mqtt.py:318:1: W391 blank line at end of file
```

**Comment by ivanv6** [comment]
> fix the flake8 failures

Hi thanks, how can i add the unit test? By pulling another request? thanks

File: test/contrib/mqtt.uts [hunk 1]
```diff
@@ -55,6 +55,9 @@ assert(connect.klive == 60)
 assert(connect.clientIdlen == 17)
 assert(connect.clientId == b'mosqpub/1440-kali')
 
+= MQTTDisconnect
+dc = MQTT()/MQTTDisconnect()
```
Comment (guedou) [inline test/contrib/mqtt.uts:59]: I think that this should be rewritten as :
```
raw = raw(MQTT()/MQTTDisconnect())
dc = MQTT(raw)
assert MQTTDisconnect in dc
```

Comment (ivanv6) [reply test/contrib/mqtt.uts]: I wrote the test you suggested but errors still remain

Comment (guedou) [reply test/contrib/mqtt.uts]: I wrote it without testing, it shoud be:
```
r = raw(MQTT()/MQTTDisconnect())                                                                                           
dc= MQTT(r)                                                                                                                
assert dc.type == 14 
```

You also need to fix the issues reported by @gpotter2

Comment (ivanv6) [reply test/contrib/mqtt.uts]: I fixed issues, still an error on the coverage but no idea on how to fix it

Comment (gpotter2) [reply test/contrib/mqtt.uts]: Coverage will be fixed if you rebase against master

Comment (ivanv6) [reply test/contrib/mqtt.uts]: @gpotter2 i run the command `git rebase master` but it returns to me `Current branch master is up to date.`.
How can I fix this?

**Pull request merged**